packages/test/index.js

```diff
@@ -0,0 +1,3 @@
+export { default } from 'enzyme-to-json';
```

Comment:
I see it very odd to export `enzyme-to-json` as the default export of `@theforeman/test`.
IMO it should be:
```js
export { default as enzymeToJson } from 'enzyme-to-json';
```